packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -98,6 +102,79 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.go(`vps.detail.dashboard.configuration.upgrade`, {
           upgradeType,
         }),
+
+      vpsMigration: /* @ngInject */ ($http, serviceName) =>
+        $http.get(`/vps/${serviceName}/migration2014`),
+
+      isMigrationRequired: /* @ngInject */ (vpsMigration) =>
+        get(vpsMigration, 'data.status') === 'toPlan',
+
+      goToVpsMigration: /* @ngInject */ (
+        $state,
+        vpsMigration,
+        catalog,
+        stateVps,
+        VpsMigrationService,
+      ) => () => {
+        const migrationPlan = VpsMigrationService.getMigrationPlan(
+          catalog,
+          vpsMigration.data.model,
+        );
+        const server = assign({
```

Comment:
Is `assign` required here? Since it is a single object, can we just do a `{ ... }`?